.github/ISSUE_TEMPLATE/config.yaml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Questions
+    url: https://forms.gle/uUhQNuPzQrvvBFJX9
+    about: Send your questions via this Google Form
```

Comment:
The `url` field does not allow for `mailto:<email>` and requires something of the format `https://`. That being the case I just created a basic Google Form that asks for the users 

1. name
2. email
3. question

If we decide not to move forward with this template I can remove it entirely and re-add it once the forum has been established. Please let me know!

Reply:
Sounds good.